app/views/admin/listings/index.html.erb

```diff
@@ -57,8 +59,19 @@
         <td><%= listing.bumped_at ? "#{time_ago_in_words(listing.bumped_at)} ago" : "Draft" %></td>
       </tr>
       <tr>
-        <td class="buffer-cell border-0" colspan="7" data-controller="buffer" data-action="load@window->buffer#autosizeBodyText">
-          <button class="btn btn-secondary" data-toggle="collapse" data-target="#buffering-area-for-listing-<%= listing.id %>" aria-expanded="false" aria-controls="buffering-area-for-listing-<%= listing.id %>">
+        <td
+          class="buffer-cell border-0"
+          colspan="7"
+          data-controller="buffer"
+          data-action="load@window->buffer#autosizeBodyText"
+          data-buffer-bg-highlighted-class="bg-highlighted"
+          data-buffer-border-highlighted-class="border-highlighted">
+          <button
+            class="btn btn-secondary"
+            data-toggle="collapse"
+            data-target="#buffering-area-for-listing-<%= listing.id %>"
```

Comment:
this is for Bootstrap, not for Stimulus :D At least now we won't mistake `data-` attributes for Bootstrap for those for Stimulus :D